### ISSUE TITLE
load classification report and confusion matrix

### ISSUE THREAD

**Issue by atikaazh** [IndoNLP/indonlu#32]
#Hi, thank you indobenchmark team for making this indobert model !!. I'm a newbie in python and excited to learn NLP. i have successfully run the "finetune_smsa.ipynb" provided in the examples folder.

Sorry, I want to ask about basic things. I confuse about how to get a classification report and confusion matrix for evaluating fitune_smsa because i don't understand the prediction variable (i mean like y_pred and y_true variable in sklearn tutorial). 

is there any syntax to get classification report and confusion matrix? Thank you in advance !!

**Comment by SamuelCahyawijaya** [comment]
Dear @atikaazh : I assume you are referring to the sklearns function [classification_report](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html) and [confusion_matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html).

You can check our [main.py](https://github.com/indobenchmark/indonlu/blob/master/main.py) file on the following lines:
https://github.com/indobenchmark/indonlu/blob/e216aab54a30d7693789d55430828f9771fd383e/main.py#L57-L61

Basically when we calculate the evaluation metrics with `metrics_fn()`, we only pass two variables which are: list of label (y_true) and list of prediction (y_pred). You can simply pass those two variables to the `classification_report` and `confusion_matrix` directly.
